dubbo-common/src/test/java/org/apache/dubbo/common/extension/ExtensionLoaderTest.java

```diff
@@ -19,11 +19,7 @@
 import org.apache.dubbo.common.Constants;
 import org.apache.dubbo.common.URL;
 import org.apache.dubbo.common.extension.activate.ActivateExt1;
-import org.apache.dubbo.common.extension.activate.impl.ActivateExt1Impl1;
-import org.apache.dubbo.common.extension.activate.impl.GroupActivateExtImpl;
-import org.apache.dubbo.common.extension.activate.impl.OrderActivateExtImpl1;
-import org.apache.dubbo.common.extension.activate.impl.OrderActivateExtImpl2;
```

Comment:
.* is not a good usage

Reply:
OK, the IDEA does this job automatically